CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 0.3.0 (2018-08-10)
+--------------------------
+
+- Adds uwsgi for production deployments.
```

Comment:
Shall we also mention that [we switched from `pykube` client to the official Kubernetes client](https://github.com/reanahub/reana-job-controller/commit/dce8aa1462e6f519671b661e89a8798b3b015b06) allowing us to upgrade REANA to latest Kubernetes?